images/linux/scripts/installers/haskell.sh

```diff
@@ -15,38 +15,57 @@ add-apt-repository -y ppa:hvr/ghc
 apt-get update
 
 # Install various versions of ghc and cabal
-apt-get install -y \
-    ghc-8.0.2 \
-    ghc-8.2.2 \
-    ghc-8.4.4 \
-    ghc-8.6.2 \
-    ghc-8.6.3 \
-    ghc-8.6.4 \
-    ghc-8.6.5 \
-    ghc-8.8.1 \
-    ghc-8.8.2 \
-    ghc-8.8.3 \
-    ghc-8.10.1 \
-    cabal-install-2.0 \
-    cabal-install-2.2 \
-    cabal-install-2.4 \
-    cabal-install-3.0 \
-    cabal-install-3.2
+if isUbuntu20 ; then
+    apt-get install -y \
+        ghc-8.6.5 \
+        ghc-8.8.3 \
+        ghc-8.10.1 \
+        cabal-install-2.4 \
+        cabal-install-3.0 \
```

Comment:
Based on #645, we should keep only the latest cabal on the image. Don't need to keep 3 versions

Reply:
Done!